src/main/java/core/basesyntax/suppliers/FigureSupplier.java

```diff
@@ -0,0 +1,57 @@
+package core.basesyntax.suppliers;
+
+import core.basesyntax.Color;
+import core.basesyntax.figures.Circle;
+import core.basesyntax.figures.Figure;
+import core.basesyntax.figures.IsoscelesTrapezoid;
+import core.basesyntax.figures.Rectangle;
+import core.basesyntax.figures.RightTriangle;
+import core.basesyntax.figures.Square;
+import java.util.Random;
+
+public class FigureSupplier {
+
+    private static final int RANDOM_NUMBER_FIGURE_RANGE = 4;
+    private static final double RANDOM_BOUND = 100.00;
+    private static final double DEFAULT_CIRCLE_VALUE = 10.0;
+    private final Random random = new Random();
+
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+
+        int randomFigureNumber = random.nextInt(RANDOM_NUMBER_FIGURE_RANGE);
+        String randomColor = colorSupplier.getRandomColor();
+        Figure randomFigure = null;
+
+        switch (randomFigureNumber) {
```

Comment:
Let's use newer version of switch-case statement like
```
return  switch (day) {
            case "Monday"-> "Week day";
            case "Tuesday"-> "Week day";
            case "Wednesday"->"Week day";
            case "Thursday"->"Week day";
            case "Friday"->"Week day";
            case "Saturday"-> "Weekend";
            case "Sunday"-> "Weekend";
            default->"Unknown";
        };
```

See the changes about this statement for different Java version [here](https://medium.com/@javatechie/the-evolution-of-switch-statement-from-java-7-to-java-17-4b5eee8d29b7)